ansible/docker.yml

```
@@ -48,5 +48,6 @@
       changed_when: "'active' not in swarm_status.stdout_lines"
     - name: join worker to swarm
       shell: >
```

This needs to be changed for the multi-line:
```suggestion
      shell: |
```